Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+PRINT 'Begin Sql scripts execution'
+SET XACT_ABORT ON
+SET NOCOUNT ON
+
+DECLARE @_acquireLock VARCHAR(64)
+DECLARE @_acquireLockMilliseconds INT
+DECLARE @_acquireLockMaxAttempts INT
+DECLARE @_longRunningThresholdMilliSeconds INT
+DECLARE @_lastAttemptsToKillSessions INT
+DECLARE @_isHosted INT
+
+PRINT 'Adding SQL files into temporary table _tmpSqlFilesTable'
+
+IF OBJECT_ID (N'_tmpSqlFilesTable', N'U') IS NOT NULL 
+BEGIN
+   PRINT 'Temporary table exists already. Dropping table _tmpSqlFilesTable'
+   DROP TABLE _tmpSqlFilesTable
+END
+CREATE TABLE _tmpSqlFilesTable (ScriptIndex INT NOT NULL IDENTITY, SqlScriptContent NVARCHAR(MAX) NULL)
+
+IF OBJECT_ID (N'_vw_tmpSqlFilesTable', N'V') IS NOT NULL 
+BEGIN
+   PRINT 'View on temporary table exists already. Dropping view _vw_tmpSqlFilesTable'
+   DROP VIEW _vw_tmpSqlFilesTable
+END
+DECLARE @_viewCommand NVARCHAR(MAX)
+SET @_viewCommand = 'CREATE VIEW _vw_tmpSqlFilesTable AS SELECT t.SqlScriptContent from _tmpSqlFilesTable t'
+EXEC sp_executesql @_viewCommand
```

Comment:
Not relevant any more. This was done because the command has to be first line in batch for SQL to compile.